worksheet_B/README.md

```diff
@@ -0,0 +1,17 @@
+```
+Display start screen
+When player presses power initiate game
+Set correct word, add 11 incorrect words and 5> 11< symbol chains
+Symbol chain is radomly set to dud or extra attempts
+Set Player attempts left to 4
+
+Loop until winning screen or player attempts = 0
+  When player selects word
+    If word is correct go to winning screen
+    Else -1 attempts
```

Comment:
The worksheet asked you to provide pseudocode for the calculation of the similarity score, i.e. loop through the words and count how many letters match between the guessed word and the secret word